components/LocationList.js

```diff
@@ -1,19 +1,26 @@
-import Card from "./Card";
+import LocationCard from "./LocationCard";
 import styled from "styled-components";
+import Link from "next/link";
+import { StyledList } from "./StyledList";
 
-const StyledList = styled.ul`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-left: 0;
+const StyledLink = styled(Link)`
+  text-decoration: none;
 `;
 
 export default function LocationList({ locations }) {
   return (
     <StyledList>
-      {locations.map((location, index) => (
-        <Card location={location} key={index} />
-      ))}
+      {locations.map((location, id) => {
+        return (
+          <StyledLink
+            href={`/locationdetail/${id}`}
+            key={id}
+            locations={locations}
```

Comment:
What is this used for?

Reply:
Hello Roland, when clicking on a location card, you are redirected to a detail page where the bee colonies are listed. That's what the link is for.

Reply:
I ment 'locations={locations}' I have no idea what it can be used for.